### PR TITLE
what's new in 24.3 beta

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://redpandadata.atlassian.net/browse/DOC-645
Review deadline: Friday, Oct 18

## Page previews
[What's New in 24.3 Beta](https://deploy-preview-811--redpanda-docs-preview.netlify.app/24.3/get-started/whats-new/)

## Checks

- [X] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/get-started/pages/whats-new.adoc [hunk 1]
```diff
@@ -5,78 +5,17 @@
...
 
-== FIPS compliance
 
-Redpanda now provides a FIPS 140-compliant binary to support running Redpanda in environments requiring FIPS-compliant software and to meet certain requirements for organizations undergoing certification through government compliance programs, such as FedRAMP. When using the FIPS binary, both Redpanda and `rpk` leverage FIPS-compliant OpenSSL libraries and ciphers for all encryption activity inside Redpanda, including encryption of data in transit. Install the `redpanda-fips` package to xref:manage:security/fips-compliance.adoc[run Redpanda in FIPS-compliance mode].
+== Mountable topics
 
-== Schema Registry support for JSON
+For topics with Tiered Storage enabled, you can xref:manage:mountable-topics.adoc[unmount a topic] to detach segment data that is still on disk to object storage, and you can mount a topic from object storage to attach the topic data to either the same origin cluster or a different one. This allows you to hibernate a topic and free up and reclaim system resources taken up by the topic, or you can migrate a topic to a different cluster.
```
Comment (kbatuigas) [inline modules/get-started/pages/whats-new.adoc]:
```suggestion
For topics with Tiered Storage enabled, you can unmount a topic to detach segment data that is still on disk to object storage, and mount that topic to either the same origin cluster, or a different one. This allows you to hibernate a topic and free up system resources taken up by the topic, or migrate a topic to a different cluster. See xref:manage:mountable-topics.adoc[Mountable topics] for details.
```
Feel free to copy edit, I'm also happy to just link to xref:manage:mountable-topics.adoc[unmount a topic] like you've done, and skip the additional "See Mountable topics" sentence.

Comment (micheleRP) [reply modules/get-started/pages/whats-new.adoc]: thanks @kbatuigas

**Comment by Deflaimun** [comment]
Properties for 24.3
https://github.com/redpanda-data/docs/pull/822

**Review by Feediver1 — APPROVED** [review]

File: modules/get-started/pages/whats-new.adoc [hunk 2]
```diff
@@ -7,6 +7,18 @@ This topic includes new content added in version {page-component-version}. For a
 * xref:redpanda-cloud:get-started:whats-new-cloud.adoc[] 
 * xref:redpanda-cloud:get-started:cloud-overview.adoc#redpanda-cloud-vs-self-managed-feature-compatibility[Redpanda Cloud vs Self-Managed feature compatibility]
 
+== Mountable topics
+
+For topics with Tiered Storage enabled, you can unmount a topic to safely detach it from a cluster and keep the topic data in the cluster's object storage bucket or container. You can mount the detached topic to either the same origin cluster, or a different one. This allows you to hibernate a topic and free up system resources taken up by the topic, or migrate a topic to a different cluster. See xref:manage:mountable-topics.adoc[Mountable topics] for details.
```
Comment (mattschumpert) [inline modules/get-started/pages/whats-new.adoc:12]: nit: should 'hibernate' be quoted?  using a metaphor here.

File: modules/get-started/pages/whats-new.adoc [hunk 3]
```diff
@@ -7,6 +7,18 @@ This topic includes new content added in version {page-component-version}. For a
...
+
+== Leader pinning
+
+For a Redpanda cluster deployed across multiple availability zones (AZs), xref:develop:produce-data/leader-pinning.adoc[leader pinning] ensures that a topic's partition leaders are geographically closer to clients. Leader pinning can lower networking costs and help guarantee lower latency by routing produce and consume requests to brokers located in certain AZs.
+
+== Debug bundle
```
Comment (mattschumpert) [inline modules/get-started/pages/whats-new.adoc]: Maybe a title like.  "Generate debug bundles from Redpanda Console"

File: modules/get-started/pages/whats-new.adoc [hunk 4]
```diff
@@ -7,6 +7,18 @@ This topic includes new content added in version {page-component-version}. For a
...
+
+You can now xref:troubleshoot:debug-bundle/index.adoc[generate a debug bundle] in Redpanda Console for comprehensive diagnostics. A debug bundle can help debug and diagnose issues with a Redpanda cluster, a broker, or the machines on which the brokers are running. You can use this file to debug issues yourself, or you can send it to the Redpanda support team to help resolve your issue.
+
 == Declarative user and ACL management in Kubernetes
 
 Redpanda now supports declarative management of users and access control lists (ACLs) using the new User resource with the Redpanda Operator. This feature allows you to:
```
Comment (mattschumpert) [inline modules/get-started/pages/whats-new.adoc]: I think the more correct technical term for 'resource' is 'Custom Resource'.
cc @david-yu

File: modules/get-started/pages/whats-new.adoc [hunk 5]
```diff
@@ -29,3 +41,22 @@ This release includes several updates to xref:get-started:licensing/overview.ado
 
 - *Unified license management in Redpanda Console*: You can now upload and apply a single license key for both Redpanda Console and the connected Redpanda cluster through the Redpanda Console UI. Any existing license key is overridden by the new one.
 
+== New commands 
+
+The following `rpk` commands are new in this version:
+
+* xref:reference:rpk/rpk-cluster/rpk-cluster-storage-cancel-mount.adoc[`rpk cluster storage cancel-mount`]
+* xref:reference:rpk/rpk-cluster/rpk-cluster-storage-list-mount.adoc[`rpk cluster storage list-mount`]
```
Comment (mattschumpert) [inline modules/get-started/pages/whats-new.adoc]: I think this may now be 'list-mountable'
@gene-redpanda to review this section.

Comment (kbatuigas) [reply modules/get-started/pages/whats-new.adoc]: @micheleRP Correct, it's list-mountable. https://github.com/redpanda-data/docs/pull/725/files#diff-3e3155706ef086bcf367fd3a3f7be815b8800e7fcc2790997fd8be35327d957bR72

File: modules/get-started/pages/whats-new.adoc (hunk 2 above)
Comment (kbatuigas) [reply modules/get-started/pages/whats-new.adoc:12]: @micheleRP we didn't put hibernate in quotes in the [doc](https://github.com/redpanda-data/docs/pull/725/files#diff-3e3155706ef086bcf367fd3a3f7be815b8800e7fcc2790997fd8be35327d957bR1) - I prefer to be consistent with not using the quotes, but I'm okay with the quotes too if that works better for What's new blurbs.

**Comment by Deflaimun** [comment]
new rpk command and some flag changes
https://github.com/redpanda-data/docs/pull/835

File: modules/get-started/pages/whats-new.adoc (hunk 5 above)
Comment (micheleRP) [reply modules/get-started/pages/whats-new.adoc]: @mattschumpert: Now we show both `rpk cluster storage list mount` and `rpk cluster storage list-mountable` [here](https://deploy-preview-811--redpanda-docs-preview.netlify.app/24.3/get-started/whats-new/#new-commands).

File: modules/get-started/pages/whats-new.adoc (hunk 4 above)
Comment (david-yu) [reply modules/get-started/pages/whats-new.adoc]: Yes that is correct custom resource https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-resources. This is for the ACL management section if that is what the comment is being referred to.

**Pull request merged**